aiven/client/cliarg.py

```diff
@@ -40,6 +40,37 @@ def wrapped(self):
     return wrapper
 
 
+def user_config_json(param_name):
```

Comment:
param_name always gets the same value. I'd just remove this parameter. This function already makes assumptions about other parameter names so might as well hard code this one too.

Reply:
`param_name` removed, parsed json is always put into destination `user_config_json`